frontend/src/pages/MissionView/MissionView.tsx

```diff
@@ -0,0 +1,111 @@
+import { generatePath, useNavigate, useParams } from "react-router-dom";
+import Button, { BUTTON_VARIANT } from "../../components/@common/Button/Button";
+import Container from "../../components/@common/Container/Container";
+import useTokenContext from "../../hooks/useTokenContext";
+import styles from "./MissionView.module.css";
+import { PARAM, PATH } from "../../constants/path";
+import { fetchMissionRequirements } from "../../api";
+import { useEffect, useState } from "react";
+import highlighter from "highlight.js";
+import "github-markdown-css/github-markdown-light.css";
+import "highlight.js/styles/github.css";
+import { Mission } from "../../../types/domains/recruitments";
+import { getMissionLabel } from "../../hooks/useMission";
+import { MISSION_STATUS } from "../../constants/recruitment";
+import { AxiosError } from "axios";
+
+const MissionView = () => {
+  const { token } = useTokenContext();
+  const navigate = useNavigate();
+
+  const { recruitmentId, missionId } = useParams<{ recruitmentId: string; missionId: string }>();
+  const [mission, setMission] = useState<Mission | null>(null);
+  const description = mission?.description ?? "";
+
+  const isMissionSubmittable =
+    !mission ||
+    mission?.submittable ||
+    mission?.status === MISSION_STATUS.SUBMITTABLE ||
+    mission?.status === MISSION_STATUS.SUBMITTING;
+
+  const missionLabel = getMissionLabel(
+    mission?.submitted ?? false,
+    mission?.status ?? MISSION_STATUS.UNSUBMITTABLE
+  );
+
+  const goBack = () => {
+    navigate(-1);
+  };
+
+  const routeToAssignmentSubmit = () => {
+    const isSubmitted = mission?.submitted;
+
+    navigate(
+      {
+        pathname: generatePath(PATH.ASSIGNMENT, {
+          status: isSubmitted ? PARAM.ASSIGNMENT_STATUS.EDIT : PARAM.ASSIGNMENT_STATUS.NEW,
+        }),
+      },
+      {
+        state: {
+          recruitmentId,
+          currentMission: mission,
+        },
+      }
+    );
+  };
+
+  const fetchRequirement = async () => {
+    try {
+      if (!recruitmentId || !missionId) {
+        goBack();
+        return;
+      }
+
+      const response = await fetchMissionRequirements({
+        token,
+        recruitmentId,
+        missionId: parseInt(missionId, 10),
+      });
+
+      setMission(response?.data);
+
+      console.log(response);
+    } catch (error) {
+      alert((error as AxiosError).response?.data.message);
+
+      goBack();
+    }
+  };
+
+  useEffect(() => {
+    fetchRequirement();
+  }, []);
+
+  useEffect(() => {
+    highlighter.highlightAll();
```

Comment:
c: highlight 대상은 어떻게 찾나요 ?-?      

[라이브러리 문서](https://highlightjs.org/)에 아래와 같이 안내되고 있긴 하군요 🤔     
> This will find and highlight code inside <pre><code> tags; it tries to detect the language automatically. If automatic detection does not work for you, you can specify the language in the class attribute:      

이후에 어떤 식으로 적용되는지 코드만으로 이해하기 어려울 수 있을 것 같아, 이 방식이 유지된다면 주석을 같이 남겨두어도 좋을 것 같습니다.